web/src/test/resources/com/epam/rft/atsy/cucumber/Positions.feature

```diff
@@ -4,52 +4,182 @@ Feature: As the user
+    Given the list is saved
+    When the user enters <title> into the title
+    And the user clicks on Mégsem button
+    Then the list of positions should left unchanged
+    Examples:
+        | title      |
+        | Új pozíció |
+        |            |
+
+  Scenario: 15 User modifies an existing position
+    #TODO Given position with Position edited title does not exist
+    When the user clicks on Edit button on a position
+    Then the title field should be filled
+    When the user enters Position edited into the title
+    And the user clicks on Mentés button
+    Then the list of positions should be changed
 
-  Scenario: user can modify an existing position
-    When the Edit button clicked on a position
-    Then the title field filled
+  # bizonyosat módosítson + azt is tesztelje le
+  #TODO Scenario Outline: 15b User modifies an existing position
+    #TODO Given position with <new_title> title does not exist
+    #TODO And position with <old_title> title exists
+    #TODO When the user clicks on Edit button on the <old_title> position
+    #TODO Then the title field should be filled
+    #TODO When the user enters <new_title> into the title
+    #TODO And the user clicks on Mentés button
+    #TODO Then the position called <old_title> should disappear from the list of positions
+    #TODO And the position called <new_title> should appear in the list of positions
+    #TODO Examples:
+    #TODO   | old_title  | new_title       |
+    #TODO   | Új pozíció | Position edited |
 
-  Scenario Outline: user cant modify the name of an existing position
-    When the Edit button clicked on a position
-    And user enters "<title>" into the title
-    And the Mentés button clicked
-    Then error message appears <message>
+  Scenario Outline: 16 User enters invalid title at position modification
+    #TODO Given position with <title> title exists
+    When the user clicks on Edit button on a position
+    And the user enters <title> into the title
+    And the user clicks on Mentés button
+    Then <message> message should appear
     Examples:
       | title     | message                |
       | Fejlesztő | Már létezik pozíció    |
+    # | Új pozíció | Már létezik pozíció "Fejlesztő" néven ! |
+
+  Scenario Outline: 17 User enters empty title at position modification
+    When the user clicks on Edit button on a position
+    And the user enters <title> into the title
+    And the user clicks on Mentés button
+    Then <message> message should appear
+    Examples:
+      | title     | message                |
       |           | Név megadása kötelező! |
 
-  Scenario: user can modify an existing position
-    When the Edit button clicked on a position
-    And user enters "Position edited" into the title
-    And the Mentés button clicked
-    Then the list of positions changed
\ No newline at end of file
+  Scenario Outline: 18 User cancels modifying an existing position
+    Given the list is saved
```

Comment:
This line seems to be there for a technical reason.

I think this line should be removed and implement the "remember existing positions" feature into the button push steps.